### ISSUE TITLE
Documentation - Networking Firewall Ports De-Mystification

### ISSUE THREAD

**Issue by pirouet** [rancher/rke2#4691]
**Is your feature request related to a problem? Please describe.**

Sort of - the issue I have is the documentation is not clear enough to identify which ports need to be open on which boxes. In a world today that is full of attacks, I don't want to leave all ports open; instead I wish to restrict ALL traffic down to only the required ports and nodes/traffic.

Through trial and error, I believe I have managed to get the correct setup but this is purely a guess on my part in the absence of anything concrete, or even example iptables.conf file to use as inspiration.

**Describe the solution you'd like**

I would like to see a visual representation of which node type needs what inbound/outbound ports taking in to account each of the different CNIs that are available out of the box.

Possibly even a sample file to use with recommendations.

**Describe alternatives you've considered**

**Additional context**

This is purely a require for documentation update as it's not 100% clear just by reading.

**Comment by rbrtbnfgl** [comment]
Hi. We have a list of used port by the various services on the RKE2 documentation. 
https://docs.rke2.io/install/requirements#networking

**Comment by pirouet** [comment]
@rbrtbnfgl  thanks but that only addresses it somewhat. For example, on the Inbound Network Rules table, how do I know who the client is calling the endpoint? I understand each setup can be different but looking at a Kubernetes example directly I would find value in something that suggested Port 9345 needs to be open on the Server nodes since the Agent needs to call in. 


If I take the first entry (port 9345), RKE2 agent nodes need to have 9345 open, but who should be able to hit it? Is 9345 coming from the Server nodes, from a LB, should it be accessible to anyone?

Does this make more sense? I'm coming from the angle of Security/Cyber to ensure traffic is only open to where it's needed. I don't get the answers from the docs

**Comment by brandond** [comment]
Yeah, we should probably have a "destination" column in that table, same as how we do in the K3s docs: https://docs.k3s.io/installation/requirements#inbound-rules-for-k3s-server-nodes

**Comment by pirouet** [comment]
@brandond Nice, yeah that's exactly what I'm thinking of! I would be able to craft iptables/nftables rules easily from that table

Thanks
